drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -161,6 +161,12 @@ def self.static_options_for_type(pick_list_type, user:)
           pluck(:OtherFunder).uniq.sort.map do |other_funder|
             { code: other_funder, label: other_funder }
           end
+      when 'WORKFLOW_DEFINITION_TEMPLATES'
+        return [] unless Hmis::Ce.configuration.enabled?
+
+        Hmis::WorkflowDefinition::Template.all.map do |template|
```

Comment:
Question about workflow definition templates -- Should they be separated across hmis data sources? Looking in the db, if I'm understanding correctly, the `owner` field is the one that would do this, but it's not enforced right now because that field is nullable. Should it be made required?